bin/apalache-mc

```diff
@@ -32,18 +32,23 @@ case `cat $DISTS | wc -l | sed 's/[[:space:]]*//g'` in
 esac
 rm -f "$DISTS"
 
+# an additional lookup list for the TLA+ modules
+TLA_LIB="$DIR/src/tla"
+
+# This is a temporary solution for handling TLA_PATH via a Java variable.
+# See apalache/#187.
+# Once tlaplus/#490,#493 are released, remove this workaround.
+test -z "$TLA_PATH" || TLA_LIB="$TLA_LIB:$TLA_PATH"
```

Comment:
However, I see that since we call the system variable `TLA-LIbrary`, there is an argument for the present naming. But, given the documentation, it seems rather that `TLA-Library` should maybe be renamed to `TLA-Path` :).

Now that I see that, I feel even less strongly about the naming here, so please ignore if you're not convinced.

Reply:
Yeah, `TLA-Library` is a feature specific to SANY: https://github.com/tlaplus/tlaplus/pull/493
`TLA_PATH` is a very new feature that hopefully will propagate in tla2tools soon.